include/matx/operators/solve.h

```diff
@@ -56,7 +56,6 @@ class SolveOp : public BaseOp<SolveOp<OpA, OpB>> {
   using matxop = bool;
   using matx_transform_op = bool;
   using solve_xform_op = bool;
-  using value_type = typename OpA::value_type;
```

Comment:
Every operator should have this defined. Did it need to be removed?

Reply:
Oh, this was auto formatted, I will put the instances back